cfn-templates/data-exports-aggregation.yaml

```diff
@@ -0,0 +1,749 @@
+  # AWS::BCMDataExports::Export outside us-east-1
+  CUR2inUSEAST1:
+    Type: Custom::CUR2Creator
+    Condition: DeployCUR2ViaLambda
+    Properties:
+      ServiceToken: !GetAtt CIDLambdaCUR2Creator.Arn
+      BucketPolicyWait: !If [ IsDestinationAccount, !Ref DestinationS3BucketPolicy, !Ref SourceS3BucketPolicy ]
+      Export:
+        DataQuery:
+          QueryStatement: !FindInMap [DataExports, CUR2, DefaultQuery]
+          TableConfigurations:
+            COST_AND_USAGE_REPORT:
+              TIME_GRANULARITY: "HOURLY"
+              INCLUDE_RESOURCES: "TRUE"
+              INCLUDE_MANUAL_DISCOUNT_COMPATIBILITY: "FALSE"
+              INCLUDE_SPLIT_COST_ALLOCATION_DATA: "FALSE"
+        Description: "CUR 2.0 export for aggregation in CID"
+        DestinationConfigurations: 
+          S3Destination:
+            S3Bucket: !If [ IsDestinationAccount, !Ref DestinationS3, !Ref SourceS3 ]
+            S3Prefix: !Sub "cur2/${AWS::AccountId}"
+            S3Region: !Ref AWS::Region
+            S3OutputConfigurations:
+              Overwrite: "OVERWRITE_REPORT"
+              Format: "PARQUET"
+              Compression: "PARQUET"
+              OutputType: "CUSTOM"
+        Name: !Ref ResourcePrefix
+        RefreshCadence:
+          Frequency: "SYNCHRONOUS"
+
+###########################################################################
+# Lambda CUR2 Creator: used to create CUR2 from outside us-east-1
+###########################################################################
+
+  CIDLambdaCUR2CreatorRole: #Execution role for the custom resource CIDLambdaAnalyticsExecutor
+    Type: AWS::IAM::Role
+    Condition: DeployCUR2ViaLambda
+    Properties:
+      Path:
+        Fn::Sub: /${ResourcePrefix}/
+      AssumeRolePolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service:
+                - lambda.amazonaws.com
+            Action:
+              - sts:AssumeRole
+      Policies:
+        - PolicyName: "ExecutionDefault"
+          PolicyDocument:
+            Version: '2012-10-17'
+            Statement:
+            - Effect: Allow
+              Action:
+              - logs:CreateLogStream
+              - logs:PutLogEvents
+              - logs:CreateLogGroup
+              Resource:
+              - !Sub "arn:${AWS::Partition}:logs:${AWS::Region}:${AWS::AccountId}:log-group:/aws/lambda/${ResourcePrefix}-CID-CUR2Creator"
+              - !Sub "arn:${AWS::Partition}:logs:${AWS::Region}:${AWS::AccountId}:log-group:/aws/lambda/${ResourcePrefix}-CID-CUR2Creator:*"
+              - !Sub "arn:${AWS::Partition}:logs:${AWS::Region}:${AWS::AccountId}:log-group:/aws/lambda/${ResourcePrefix}-CID-CUR2Creator:*:*"
+        - PolicyName: "ExecutionSpecific"
+          PolicyDocument:
+            Version: '2012-10-17'
+            Statement:
+            - Effect: Allow
+              Action:
+              - cur:PutReportDefinition
```

Comment:
do we need it for CUR2?

Reply:
Yes, `cur:PutReportDefinition` is required for creating a CUR 2.0 in Data Exports. This was implemented to prevent CUR 2 from becoming a backdoor way to access CUR data. For example, if a management account had an SCP blocking `cur:PutReportDefinition` and then we launched CUR 2, they could get access to those APIs and create a CUR.